vosekast_control/Vosekast.py

```diff
@@ -317,12 +317,12 @@ def clean(self):
                     if command['command'] == 'close':
                         valve.close()
                         return
-                    if command['command'] == 'open':
+                    elif command['command'] == 'open':
                         valve.open()
                         return
                     else:
                         self.logger.warning(
-                            f'target_id found. command {command["command"]} did not execute.')
+                            f'did not found target_id {target_id}. command {command["command"]} did not execute.')
```

Comment:
if it gets there, it has already passed the 
```
if valve.name == target_id:
```
check. therefore 
```
self.logger.warning(f'target_id found. command {command["command"]} did not execute.')
```
should be correct, but there was no matching command message.

Reply:
correct i fixed that